src/renderer/services/docker/images.ts

```diff
@@ -0,0 +1,51 @@
+import {SupportedNetworks} from "../eth2/supportedNetworks";
+import {BeaconChain} from "./chain";
+import {IDockerRunParams} from "./type";
+
+const eth1Endpoint = "https://goerli.infura.io/v3/9b8caef145c74574869579199c47e847";
+
+export const getLighthouseBeaconChainConfig = (network: SupportedNetworks): IDockerRunParams => ({
```

Comment:
That's not really beacon chain config but rather docker config. I would maybe integrate this in networks configuration, but split into different files

Reply:
Like:
```
/services/networks
   /index.ts with network cofniguration
   /schlesi.ts
   /local.ts
   etc
```